src/application/Layouts/FullWidthLayout.tsx

```diff
@@ -0,0 +1,65 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/core";
+import { useTheme } from "emotion-theming";
+import { rem } from "polished";
+import { Navigation, NavigationProps } from "./Navigation";
+
+interface FullWidthLayoutProps {
+  navigationProps: NavigationProps
+  children: any;
+};
+
+interface FullWidthLayoutComposition {
+  Header: React.FC;
+  Main: React.FC;
+}
+
+const layout = css`
```

Comment:
Coding preference question: do you think we should name our serialized emotion styles with a style suffix of some kind, to differentiate them a bit from other vars in our source files? E.g. instead of `layout` we use something like `layoutStyles` or `layoutCss`, just to differentiate them a bit at a quick glance from other non-css based vars.

Reply:
Oh, yes! That's a great idea. I'll update soon-ish.